index.md

```diff
@@ -12,9 +12,12 @@ open source innovation projects within the energy and electricity sectors.
 
 # Open source
 Powsybl in an open source framework licensed under the [Mozilla Public License 2.0](https://www.mozilla.org/en-US/MPL/2.0/).
-The source code is hosted on GitHub in two different repositories:
+The source code is hosted on GitHub in several repositories:
 - [powsybl-core](https://github.com/powsybl/powsybl-core) contains all the core features
+- [powsybl-math-native](https://github.com/powsybl/powsybl-math-native) are natives modules (JNI) used by powsybl-math modules (LU factorization)
```

Comment:
May in more details? "are native module (C++ wrapped with JNI compiled for multiple platforms and and delivered in jars)"